### DISCUSSION TITLE
July 2024 Conflux Documentation Contributor Incentives

### DISCUSSION THREAD

**Discussion by darwintree** [Conflux-Chain/conflux-documentation#693] [Announcements]
We are excited to announce the eligibility list of contributors who are qualified for contributor incentives for 202407. We deeply appreciate your support and highly value the insights and expertise you bring to our community.

以下为202407有资格获取激励的贡献者名单。非常感谢名单中的各位对本项目的支持。

The following GitHub contributors are eligible:

以下的GitHub贡献者有资格领取激励：

* [SPCFXDA](https://github.com/SPCFXDA)
* [0xffffb](https://github.com/0xffffb)
* [jackleeio](https://github.com/jackleeio)
* [lululu00](https://github.com/lululu00)
* [wongzigii](https://github.com/wongzigii)

The eligibility list is curated in accordance with our [CONTRIBUTING guideline](https://doc.confluxnetwork.org/docs/general/CONTRIBUTING). If you believe your contribution has been overlooked, please do not hesitate to contact us.

名单是根据[CONTRIBUTING指导原则](https://doc.confluxnetwork.org/docs/general/CONTRIBUTING)所挑选的。如果您有对应贡献但被遗漏在了名单外，欢迎联系我们。

## Contributor Incentives Details 贡献者激励详情

We have introduced financial incentives for contributors which are **directly** related to the quality of the contributor's work.

我们为仓库的贡献者引入的经济激励是与贡献者的工作质量所**直接**相关的。

For incentives for documentation content contributors, please refer to:

https://confluxglobal.notion.site/de1167fff4ec4bc299f826dc8474d511?v=ae21d522a235474c80c09bca2a0e0393&pvs=74

and their incentives are calculated according to the [2024Q2 reward guidelines](https://github.com/Conflux-Chain/conflux-documentation/blob/2024Q1-reward/docs/general/CONTRIBUTING/reward-guidelines.md). If you have relevant contributions but have been omitted from the list, please feel free to contact us. In addition, we will collect the (temporary) addresses provided by the relevant contributors for airdrop redemption in this discussion.

文档内容贡献者的激励请参考：

https://confluxglobal.notion.site/de1167fff4ec4bc299f826dc8474d511?v=ae21d522a235474c80c09bca2a0e0393&pvs=74

其激励根据[2024Q2 reward guidelines](https://github.com/Conflux-Chain/conflux-documentation/blob/2024Q1-reward/docs/general/CONTRIBUTING/reward-guidelines.md)计算。如果您有对应贡献但被遗漏在了名单外，欢迎联系我们。此外我们将在本discussion中收集相关贡献者提供的领取空投的（临时）地址。

Thank you for your assistance in enhancing our project. If you have any queries or require assistance, please feel free to create an issue in our Github repository.

感谢各位持续帮助我们改进项目。如果你还有任何疑问或需要帮助，请随时在我们的Github仓库中创建issue。

**Comment by darwintree** [comment]
@SPCFXDA @0xffffb @wongzigii  @shengchenshengchen 
Thank you for your contribution! We would know your address on Conflux core space for the airdrop

**Comment by SPCFXDA** [comment]
Hi,
My Conflux core address is : cfx:aap3r3huv53ne832guk5j3pykr0rfj15yehn2gcspt

Thanks
SP

**Comment by wongzigii** [comment]
cfx:aajnkjjfuvkwy7jkxusymy85wdn549fxd6m013gyu0 

Thanks
